<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MOVIX</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtresource.qrc">
    <normaloff>:/png/icon.png</normaloff>:/png/icon.png</iconset>
  </property>
  <widget class="QComboBox" name="fileTypeCombo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Save Format</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>html</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>csv</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>385</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="crtInfo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>regata</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color : #6ee6c7; } </string>
   </property>
   <property name="text">
    <string>Watchlist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>115</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Show Watchlist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="watchedBtn">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>115</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Seen</string>
   </property>
  </widget>
  <widget class="QComboBox" name="genreComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>165</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Select Genre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>all</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="yesRadio">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>205</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="NoRadio">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>235</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>180</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Did you like it?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openHTML_Btn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>385</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Open HTML</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openCSV_Btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>385</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Open CSV</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>115</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { color : #6ee6c7; } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qtresource.qrc">
     <normaloff>:/png/add.png</normaloff>:/png/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="viewBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qtresource.qrc">
     <normaloff>:/png/play.png</normaloff>:/png/play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="userLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color : #6ee6c7; } </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/png/icon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qtresource.qrc"/>
 </resources>
 <connections/>
</ui>
